### ISSUE TITLE
[Bug]: After some navigation the content of the left sidebar spills

### ISSUE THREAD

**Issue by rhizoome** [KosmosisDire/obsidian-webpage-export#549]
### What happened?

<img width="489" alt="image" src="https://github.com/user-attachments/assets/fc4711dd-5fea-4eca-903e-46295adeae43">

I clicked a few links and suddenly the left sidebar spilled. Open and closing it does not help.

### Reproduction

Works every time. Tested on mobile iOS and responsive mode in Safari.

### Last Working Version

1.9.0.2b

### Version

1.9.0.3b

### Operating System

iOS 17/18 | macOS 15

### Obsidian Version

1.7.7

### Which browsers can you reproduce this in?

Safari, Safari IOS

### Relevant log output

```yaml
na
```

**Comment by KosmosisDire** [comment]
Yup I noticed this when testing your last bug report, it is again a safari specific issue I'll have to look into. *Sigh* lol

**Comment by rhizoome** [comment]
Yea, Safari I was having trouble with some of my CSS snippets and it made trouble the last 20 years.
